wgpu-hal/src/gles/egl.rs

```diff
@@ -1,8 +1,11 @@
 use glow::HasContext;
-use parking_lot::Mutex;
+use parking_lot::{Mutex, MutexGuard};
 
 use std::{ffi::CStr, os::raw, ptr, sync::Arc};
 
+/// The amount of time to wait while trying to obtain a lock to the adapter context
+const CONTEXT_LOCK_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(1);
```

Comment:
let's be consistent with other `std` includes by including either `std::time` or `std::time::Duration`